client/scss/utilities/_root-scope-classes.scss

```diff
@@ -293,6 +290,24 @@
   position: absolute;
 }
 
+.half-in-image {
+  margin-top: -50px;
+}
+
+.show-to-l {
+  @include respond-to('large') {
+    display: none;
+  }
+}
+
+.person-image {
```

Comment:
also bespoke/not very root-scopey

Reply:
Yea, didn't really know what to do with this puppy.

Reply:
I'll ask @Norvard about whether we'll have a contributor thing that will be used elsewhere.